src/open-api/from-open-api.ts

```diff
@@ -43,7 +50,15 @@ export async function fromOpenApi(
         continue
       }
 
-      const operation = pathItem[method] as OpenAPIV3.OperationObject
+      const rowOperation = pathItem[method] as OpenAPIV3.OperationObject
+      if (!rowOperation) {
+        continue
+      }
+
+      const operation = mapOperation
+        ? mapOperation(url, method, rowOperation)
```

Comment:
I think this is properly referred to as `path`, not `url`, is it? We may consider renaming it across the implementation, including the new map function.

Reply:
yea path is the open api spec concept. Renamed in the mapper function and also other places that touches open api spec: https://github.com/mswjs/source/pull/57/commits/c642e1ff2a8aabb4ca4acb68af6d2059ac6ac7c0

If I missed `url` to `path` renaming somewhere else, let me know. I did not touch places such as `isAbsoluteUrl` since that one has to do with url as per my understanding and not the open api spec path concept